src/examples/train-delay-repay/package.json

```diff
@@ -0,0 +1,11 @@
+{
+  "name": "places-list",
```

Comment:
Probably not `places` ;)

Reply:
Fixed along with other copy-pasted `package.json` files :)